libtock/tock.c

```diff
@@ -40,6 +40,12 @@ void yield_for(bool *cond) {
   }
 }
 
+#if defined(__thumb__)
+
+//
+// IMPLEMENATTION FOR CORTEX-M THUMB BASED PLATFORMS
```

Comment:
```suggestion
```

Cosmetic / consistency thing: I think these comments are redundant with the `if defined` clause above -- if you really think they're necessary then they should probably be added to the crt0 as well

edit: the lack of a multi-line suggestion in github is criminal

Reply:
Can you fix the typo in `implementation` too? :P